components/organisms/o-microcart-panel.vue

```diff
@@ -20,8 +20,8 @@
               :key="product.id"
               :image="getThumbnailForProductExtend(product)"
               :title="product.name"
-              :regular-price="getProductPrice(product).regular"
-              :special-price="getProductPrice(product).special"
+              :regular-price="!!getProductPrice(product).regular ? formatPrice(getProductPrice(product).regular) : null"
```

Comment:
```suggestion
              :regular-price=" formatPrice(getProductPrice(product).regular)"
```
I think it should work fine